invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/search.html

```diff
@@ -55,7 +55,7 @@
   "searchApi": {
     "axios": {
       "headers": {
-        "Accept": "application/json"
+        "Accept": "application/vnd.rdmv1+json"
```

Comment:
Comment: Just needs to be aligned to a change i requested in invenio-rdm-records